core/src/main/java/org/testcontainers/images/RemoteDockerImage.java

```diff
@@ -79,6 +80,7 @@ protected final String resolve() {
                         .withTag(imageName.getVersionPart())
                         .exec(new TimeLimitedLoggedPullImageResultCallback(logger))
```

Comment:
Thinking about it, I'm inclined to move some parts of this into `LoggedPullImageResultCallback`. Will move PR back to WIP status and do it.

Reply:
Done